frontends/web/src/components/view/view.tsx

```diff
@@ -23,20 +23,31 @@ import style from './view.module.css';
 
 type ViewProps = {
     dialog?: boolean;
+    fitContent?: boolean;
     fullscreen?: boolean;
     minHeight?: string;
-    top?: boolean;
     onClose?: () => void;
     position?: 'fill' | '';
     textCenter?: boolean;
     width?: string;
     withBottomBar?: boolean;
 }
 
+/**
+ * View component is used as a container component to wrap ViewHeader, ViewContent and ViewButtons
```

Comment:
did we use this docstring format in the past or is it the first time?

Reply:
We used it in the past

![Screen Shot 2022-06-07 at 12 17 24 PM](https://user-images.githubusercontent.com/546900/172356556-b1a390af-dda5-4a74-b891-638d7235c3d2.png)

Reply:
https://github.com/digitalbitbox/bitbox-wallet-app/blob/2686682a73e28c526cbd9a80beab7e6430f174c6/frontends/web/src/decorators/subscribe.tsx#L25-L32